components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1444,6 +1484,40 @@ void RewardsServiceImpl::SetConfirmationsIsReady(const bool is_ready) {
     ads_service->SetConfirmationsIsReady(is_ready);
 }
 
+void RewardsServiceImpl::OnGetAdsNotificationsHistoryMojoProxy(
```

Comment:
not sure why this has `MojoProxy`, but I think it doesn't matter because it will go away anyway and be replaced by a callback in `GetAdsNotificationsHistory`